storage/database/db_manager.go

```diff
@@ -627,6 +608,25 @@ func (dbm *databaseManager) DeleteHeader(hash common.Hash, number uint64) {
 	dbm.cm.deleteHeaderCache(hash)
 }
 
+// Head Number operations.
+// ReadHeaderNumber returns the header number assigned to a hash.
+func (dbm *databaseManager) ReadHeaderNumber(hash common.Hash) *uint64 {
```

Comment:
Is this function just a location change?

Reply:
@jimni1222 Right. I moved the location of `DBManager.ReadHeaderNumber ` in order to make it align with the change at https://github.com/klaytn/klaytn/pull/121/files#diff-35f1b0eff63fa77674c3fe3368e7f795R64